lib/cdo/script_constants.rb

```diff
@@ -51,14 +56,43 @@ def ScriptConstants.hoc?(name)
         MINECRAFT_NAME,
         INFINITY_NAME,
         ARTIST_NAME
```

Comment:
Talked to Tanya, she just wanted me to confirm that the `/api/hour/begin` and `/api/hour/end` links still work and Activities are still added to the db. Both work, even after the `script_in_category?` refactor.